app/controllers/api/avatars.php

```diff
@@ -635,6 +637,243 @@
             ->file($image->getImageBlob());
     });
 
+App::get('/v1/avatars/screenshots')
+    ->desc('Get webpage screenshot')
+    ->groups(['api', 'avatars'])
+    ->label('scope', 'avatars.read')
+    ->label('cache', true)
+    ->label('cache.resourceType', 'avatar/screenshot')
+    ->label('cache.resource', 'screenshot/{request.url}/{request.width}/{request.height}/{request.scale}/{request.theme}/{request.userAgent}/{request.fullpage}/{request.locale}/{request.timezone}/{request.latitude}/{request.longitude}/{request.accuracy}/{request.touch}/{request.permissions}/{request.sleep}/{request.quality}/{request.output}')
```

Comment:
Not sure if we have base rate limt, and what is value. To be extra safe, I would mention abuse labels

Reply:
updated with abuse-limit